src/Facebook/InstantArticles/Transformer/Getters/ElementGetter.php

```diff
@@ -53,7 +53,8 @@ public function withSelector($selector)
     public function get($node)
     {
         $elements = self::findAll($node, $this->selector);
-        if (!empty($elements) && property_exists($elements, 'length') && $elements->length !== 0) {
+        // var_dump($elements instanceof DOMNodeList);
+        if (!empty($elements) /*&& property_exists($elements, 'length')*/ && $elements->length !== 0) {
```

Comment:
We need to be sure the element has the property "length" and we should be sure it is a number and not zero (needs to be positive number).

Reply:
The problem here is that property_exists doesn't work with HHVM DOMNodeList, it looks like it's creating the property dynamically so it always returns false. Is there anything else we can check? When this might not come with length?